moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -214,6 +220,12 @@ class Servo
   std::unique_ptr<CollisionMonitor> collision_monitor_;
 
   pluginlib::UniquePtr<online_signal_smoothing::SmoothingBaseClass> smoother_ = nullptr;
+
+  moveit::core::RobotStatePtr robot_state_;
+  const moveit::core::JointModelGroup* joint_model_group_;
+  std::vector<std::string> joint_names_;
+
+  bool invalid_parameter_update_;
```

Comment:
I don't think we need this if it's used as synonymous with servo being paused.

I also can't find that line in the PR review, but I think your changes make you able to remove the `servo_paused_` member variable as well?